models/articles.models.js

```diff
@@ -89,4 +89,18 @@ exports.fetchAllArticles = () => {
                         return articles.rows;
                      });
                   });
-               };
+};
+
+exports.fetchCommentByArticleId = (article_id) => {
+  return db.query(`SELECT * FROM comments 
+  WHERE article_id = $1
+  ORDER BY created_at DESC`, [article_id]).then((comments) => {
+    if (comments.rows.length === 0) {
+      return Promise.reject({
+        status: 404,
```

Comment:
How would this handle if the id existed but there just wasn't any comments yet? 

Would you want that to reject?